diff --git a/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.mm b/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.mm
index 1bc361f..21d8e59 100644
--- a/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.mm
+++ b/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.mm
@@ -32,7 +32,7 @@
   if (self.reactDelegate != nil) {
     return [((id<EXReactDelegateProtocol>)self.reactDelegate) createReactRootViewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions];
   }
-  return [super viewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions devMenuConfiguration:devMenuConfiguration];
+  return [super viewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions];
 }
 
 - (UIView *)superViewWithModuleName:(NSString *)moduleName
@@ -40,11 +40,7 @@
                       launchOptions:(nullable NSDictionary *)launchOptions
                devMenuConfiguration:(nullable RCTDevMenuConfiguration *)devMenuConfiguration
 {
-  if (devMenuConfiguration == nil) {
-    devMenuConfiguration = [RCTDevMenuConfiguration defaultConfiguration];
-  }
-
-  return [super viewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions devMenuConfiguration:devMenuConfiguration];
+  return [super viewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions];
 }
 #else
 - (UIView *)viewWithModuleName:(NSString *)moduleName
diff --git a/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift b/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift
index 925e0f9..e43f627 100644
--- a/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift
+++ b/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift
@@ -132,8 +132,7 @@ public class ExpoReactNativeFactory: ExpoReactNativeFactoryObjC, ExpoReactNative
       rootView = rootViewFactory.view(
         withModuleName: moduleName ?? defaultModuleName,
         initialProperties: initialProps,
-        launchOptions: launchOptions,
-        devMenuConfiguration: self.devMenuConfiguration
+        launchOptions: launchOptions
       )
 #else
       rootView = rootViewFactory.view(
